install-macos.sh

```diff
@@ -93,7 +93,11 @@ CONFIG="{\"ACTIVATION_REQUESTED_NONCE\":$ACTIVATION_REQUESTED_NONCE,\"AGENT_KEY\
 echo "$CONFIG" | $SUDO tee "$VANTA_CONF_PATH" > /dev/null
 $SUDO /bin/chmod 600 "$VANTA_CONF_PATH"
 $SUDO /usr/sbin/chown root:wheel "$VANTA_CONF_PATH"
-$SUDO /usr/sbin/installer -pkg $PKG_PATH -target / >/dev/null
+$SUDO /usr/sbin/installer -pkg $PKG_PATH -target /
+# temporary log installation logs
+tail -n 50 /var/log/install.log
```

Comment:
`set -e` is preventing this line from getting hit!

There should also be a -verbose option on `installer` :)

Reply:
Thank you 🙏